File: blog/creator.py
class Creator(object):
    """

    """

    def __init__(self, id, username, name, sourceUrl):
        self.id = id
        self.username = username
        self.name = name
        self.sourceUrl = sourceUrl

    def toJSON(self):
        return self.__dict__

    @staticmethod
    def fromJSON(json):
        return Creator(
            json.get("id"),
            json.get("username"),
            json.get("name"),
            json.get("sourceUrl")
        )

File: packages/js/lib/creator.js
import {Record} from "immutable";

export class Creator extends Record({
    id: null,
    username: null,
    name: null,
    sourceUrl: null,
    imageUrl: null
}) {
    static fromJS(js) {
        return new Creator(js);
    }

    static fromJSON(json) {
        return Creator.fromJS(json);
    }
}

export default Creator;
